hw/drivers/sensors/lps33thw/src/lps33thw_priv.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * lps33thw_priv.h
+ *
+ *  Created on: Aug 21, 2018
+ *      Author: justine
+ */
+
+#ifndef REPOS_APACHE_MYNEWT_CORE_HW_DRIVERS_SENSORS_LPS33THW_SRC_LPS33THW_PRIV_H_
+#define REPOS_APACHE_MYNEWT_CORE_HW_DRIVERS_SENSORS_LPS33THW_SRC_LPS33THW_PRIV_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum lps33thw_registers {
+    LPS33THW_INTERRUPT_CFG = 0x0b,
+    LPS33THW_THS_P = 0x0c,
+    LPS33THW_IF_CTRL = 0x0e, //added
+    LPS33THW_WHO_AM_I = 0x0f,
+    LPS33THW_CTRL_REG1 = 0x10,
+    LPS33THW_CTRL_REG2 = 0x11,
+    LPS33THW_CTRL_REG3 = 0x12,
+    LPS33THW_FIFO_CTRL = 0x13, //diff reg add
+    LPS33THW_FIFO_WTM = 0x14, //added
+    LPS33THW_REF_P = 0x15,  //no ref_p_h
+    LPS33THW_RPDS = 0x18,
+    LPS33THW_INT_SOURCE = 0x24, //diff reg add
+    LPS33THW_FIFO_STATUS1 = 0x25, //diff reg add
+    LPS33THW_FIFO_STATUS2 = 0x26, //added
+    LPS33THW_STATUS = 0x27,
+    LPS33THW_PRESS_OUT = 0x28,
+    LPS33THW_TEMP_OUT = 0x2b,
+    LPS33THW_FIFO_SDATA_OUT_PRESS = 0x78, //added
+    LPS33THW_FIFO_SDATA_OUT_TEMP = 0x7b, //added
```

Comment:
C++ style comments are not allowed as per the coding style guide. BTW, the comments don't give enough info about the field.